### DISCUSSION TITLE
Using Conventional Commits

### DISCUSSION THREAD

**Discussion by iancooper** [BrighterCommand/Brighter#1393] [Q&A]
I'm never the best at commit messages, and I think I would benefit from structure.

Conventional Commits is an approach used by some to structure commit messages: https://www.conventionalcommits.org/en/v1.0.0/

it has advantages that if followed, tooling can look for how to create version numbers etc.

We would likely want to enforce in our GHA build pipeline using something like: https://commitsar.tech/docs/usage/github

Thoughts? @holytshirt ?

**Comment by iancooper** [comment]
@holytshirt I suggest we turn on post getting 9 out of the door, so we can pick up the new process after the push to get this work done?

**Reply by holytshirt** [reply]
Yeah, I've been try a few commits out with it to get a feel.
It forces you to think about it which is always a good thing
